doc/CHANGES.md

```diff
@@ -110,8 +113,17 @@
     app/views/admin_general/index.html.erb
     app/views/comment/_single_comment.html.erb
     app/views/followups/_followup.html.erb
+    app/views/general/_responsive_header.html.erb
+    app/views/layouts/admin.html.erb
+    app/views/layouts/default.html.erb
+    app/views/layouts/no_chrome.html.erb
     app/views/reports/new.html.erb
     app/views/request/describe_notices/_error_message.html.erb
+    app/views/request/describe_notices/_internal_review.html.erb
+    app/views/request/describe_notices/_not_held.html.erb
+    app/views/request/describe_notices/_successful.html.erb
+    app/views/request/describe_notices/_waiting_response.html.erb
+    app/views/request/describe_notices/_waiting_response_overdue.html.erb
```

Comment:
No – newly created ones wouldn't really be overridden so reusers wouldn't need to reconcile changes.